src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IPublicationRepository.cs

```diff
@@ -17,6 +17,8 @@ public interface IPublicationRepository
 
         Task<MyPublicationViewModel> GetPublicationWithAllReleases(Guid publicationId);
 
+        Task<List<Release>> GetLatestReleases(Guid publicationId);
```

Comment:
I originally named this `GetLatestVersionsOfAllReleases` and then seem to renamed it for some reason - I cannot see a PR comment asking me to change it, so not sure why I changed it again. 

I'll switch it back to `GetLatestVersionsOfAllReleases` - I think it needs to be plural `Versions` to say that it can be more than one. But happy to go with something else if you'd prefer.